src/apiClients/AirTableApiClient.ts

```diff
@@ -41,6 +48,12 @@ interface Record {
   };
 }
 
+/**
+ * Takes raw data from AirTable API call and stores it in a Record object.
+ *
+ * @param record - Record object containing one row's data (corresponding to one organization) from AirTable
+ * @returns A new Organization object containing data from the record parameter
+ */
```

Comment:
Should we be using double square brackets around references to objects?

From Notion: "The [[]] brackets indicate a link to a different class (you don't have to use them but if your class relates to another part of the code, then please try it out)."